### ISSUE TITLE
Form Attributes and Accessibility Enhancements

### ISSUE THREAD

**Issue by SeanMarkWD** [SeanMarkWD/Travel-Agency-Landing-Page#18]
Required aria-label missing from INPUT fields:
Select destination
From
Enter a number of guests

**Comment by SeanMarkWD** [comment]
Added aria-label to  INPUT fields:
Select destination
From
Enter a number of guests
